epictrack-web/src/components/user/UserList.tsx

```diff
@@ -36,7 +36,12 @@ const UserList = () => {
     try {
       const userResult = await UserService.getUsers();
       if (userResult.status === 200) {
-        setUsers(userResult.data as never);
+        const sortedResults = (userResult.data as User[]).sort((a, b) => {

```

Comment:
Do you really need Infinity here. Can't it just work with zero

Reply:
we can do 0